server/raft.go

```diff
@@ -892,9 +894,9 @@ func (n *raft) ProposeAddPeer(peer string) error {
 func (n *raft) doRemovePeerAsLeader(peer string) {
 	n.Lock()
 	if n.removed == nil {
-		n.removed = map[string]struct{}{}
+		n.removed = map[string]int64{}
 	}
-	n.removed[peer] = struct{}{}
+	n.removed[peer] = time.Now().UnixNano()
```

Comment:
Could use new access time here.

Reply:
That feels like a pre-optimization, and don't really think it's needed. This only gets run once when a peer is removed, which shouldn't be happening often enough to need to use the new `getAccessTime()` in my opinion?

Reply:
Not pre since we know this can affect the system (UnixNano).. You could make the map take a time vs int64, those are cheap..

Also we can not control how may times peer remove can be called per second atm.